app/install/aws-cloudformation.md

```diff
@@ -120,8 +92,7 @@ Provisions Kong resources with new or user provided Postgres DB in a new VPC or
 
 4. **Parameters**:
 
-    Fill in all the parameters details. If you chose to launch Kong with Cassandra/Postgres you would be asked to fill in extra parameters to create a Cassandra cluster or Postgres RDS instance. 
-    check the description of each field and provide appropriate values.
+    Fill in all the parameter details. If you chose to launch Kong with a new Postgres instance, you will be asked to fill in extra parameters to create a Postgres RDS instance. Check the description of each field and provide appropriate values.
```

Comment:
I think the s in "parameters" was appropriate.
"and provide *the* appropriate values"